twake/backend/node/config/default.json

```diff
@@ -139,6 +143,20 @@
     }
   },
   "services": [
+    "auth",
+    "counter",
+    "database",
+    "phpnode",
+    "pubsub",
+    "push",
+    "realtime",
+    "search",
+    "storage",
+    "tracker",
+    "webserver",
+    "websocket"
+  ],
+  "_services": [
```

Comment:
Yes, it should be removed

Reply:
unfortunately we need all services here for now, after I change the way web services are detected it will be necessary to specify only the platform services here